src/tv2_afvd_showstyle/__tests__/baseline.spec.ts

```diff
@@ -23,16 +23,18 @@ const SEGMENT_ID = 'test_segment'
 const PART_ID = 'test_part'
 
 describe('Baseline', () => {
-	test('Config: ' + configSpec.id, () => {
+	test('Config: ' + configSpec.id, async () => {
 		expect(configSpec.studioConfig).toBeTruthy()
 		expect(configSpec.showStyleConfig).toBeTruthy()
 
 		const mockRundown: ExtendedIngestRundown = createMockRundown()
-		const mockContext: ShowStyleUserContext = createMockContext(mockRundown.name)
+		const mockContext: GetRundownContext = createMockContext(mockRundown.name)
 
-		const result = Blueprints.getRundown(mockContext, mockRundown)
+		const result = await Blueprints.getRundown(mockContext, mockRundown)
+		if (result === null) {
+			fail('Result is not allowed to null')
```

Comment:
Did not know about fail not being part of Jest. But yeah, seems like we should starting migrating away from fail.
Changed to be throw an exception instead.
(I don't like the expect.assertions() approach since we won't get any meaningfull error messages from that)